shapeshift/web
# chore: bump shapeshiftoss packages

## Description



* bump all `@shapeshiftoss/*` packages
* **allows adding ethereum accounts with native wallet**

## Notice




## Pull Request Type


## Issue (if applicable)



closes #2540 

## Risk

minimal and should largely be isolated to hdwallet changes in https://github.com/shapeshift/hdwallet/pull/571



## Testing



* connect native wallet
* turn on multi accounts flag
* you should now be able to add an ethereum account with the native wallet

### Engineering



* see common testing steps

### Operations



* see common testing steps

## Screenshots (if applicable)

<img width="1251" alt="image" src="https://user-images.githubusercontent.com/88504456/190455825-3868eb48-7188-4de1-8412-274d9254babb.png">
